robottelo/hosts.py

```diff
@@ -1550,6 +1563,17 @@ class Capsule(ContentHost, CapsuleMixins):
     product_rpm_name = 'satellite-capsule'
     upstream_rpm_name = 'foreman-proxy'
 
+    def __init__(self, hostname, **kwargs):
+        kwargs.setdefault('net_type', settings.capsule.network_type)
+        super().__init__(hostname=hostname, **kwargs)
+
+    @property
+    def network_type(self):
+        """Get the network type of the host"""
+        if not self._net_type:
+            self._net_type = HostNetworkType(settings.capsule.network_type)
+        return self._net_type
+
```

Comment:
Not sure which one of these we should keep. :thinking:

Reply:
only the `setdefault` one as the net_type kwarg is handled in ContentHost's init. If we set `net_type` in Capsule init, it is going to cascade to ContentHost's init.